data-new/TestAndVerification/Consolidation.yaml

```diff
@@ -1,16 +1,17 @@
----
 _yaml_references:
   tools:
     ci-cd: &ci-cd CI/CD tools, eg. Jenkins
-    containers: &containers Container technologies and orchestration like Docker, Kubernetes
+    containers: &containers Container technologies and orchestration like Docker,
+      Kubernetes
     dependabot: &dependabot >-
       [dependabot](https://dependabot.com/)
     defect-dojo: &defect-dojo >-
       [OWASP Defect Dojo](https://github.com/DefectDojo/django-DefectDojo)
     purify: &purify https://github.com/faloker/purify
-    owasp-dependency-check: &owasp-dependency-check OWASP Dependency Check Jenkins Plugin
+    owasp-dependency-check: &owasp-dependency-check OWASP Dependency Check Jenkins
+      Plugin
     logparser-jenkins: &logparser-jenkins LogParser Jenkins Plugins
-    mozilla-zest: &mozilla-zest   https://github.com/mozilla/zest
+    mozilla-zest: &mozilla-zest https://github.com/mozilla/zest
```

Comment:
Is this intentional or should we remove the link?

Reply:
The link is ok.